app/models/category.rb

```diff
@@ -1,5 +1,8 @@
 class Category < ApplicationRecord
-  CATEGORY_PARAMS = %i(name).freeze
+  CATEGORY_PARAMS = %i(name status).freeze
+  enum status: {not_show: 0, show: 1}
 
   has_many :newspapers, dependent: :destroy
+
+  scope :by_status_show, -> { where status: 1}
```

Comment:
```suggestion
  scope :by_status_show, -> {where status: :not_show}
```
dùng key enum thay vì số nhé